cypress/fixtures/articles_data.json

```diff
@@ -0,0 +1,33 @@
+{
+  "articles": [
+    {
+      "id": 1,
+      "title": "Title1",
+      "subheader": "subheader 1",
+      "image_path": "https://images.unsplash.com/photo-1489710437720-ebb67ec84dd2?ixid=MXwxMjA3fDB8MHxzZWFyY2h8NTZ8fGhhcHB5JTIwcGVvcGxlfGVufDB8fDB8&ixlib=rb-1.2.1&auto=format&fit=crop&w=800&q=60",
```

Comment:
are you getting image from the backend?

Reply:
No, not at the moment! We did this just for testing and styling purpose.